Analysis/DEEP2_R23_O32/Plotting/more_plots.py

```diff
@@ -250,23 +260,45 @@ def dust_att_plot(combine_flux):
     #pdf_pages.close()
 
     
-def plotting_individual_for_stacking_image():
-    name = '/Users/reagenleimbach/Desktop/Zcalbase_gal/individual_plots_for_stacking_image.pdf'
-    pdf_pages = PdfPages(name)
-    
-    RestframeMaster = r'/Users/reagenleimbach/Desktop/Zcalbase_gal/Master_Grid.fits'
+def plotting_individual_for_stacking_image(stack_spectra=False):
+    if not stack_spectra:
+        name = '/Users/reagenleimbach/Desktop/Zcalbase_gal/individual_plots_for_stacking_image.pdf'
+        RestframeMaster = r'/Users/reagenleimbach/Desktop/Zcalbase_gal/Master_Grid.fits'
+        spec_range = range(100, 120)
+        y_lim = (-2,2)
+        y_text = 1.95
+        left = 0.13
+    else:
+        RestframeMaster = r'/Users/reagenleimbach/Desktop/Zcalbase_gal/R23O32_Manual_0417/Stacking_Masked_MasterGrid_n_Bins.fits'
+        name = '/Users/reagenleimbach/Desktop/Zcalbase_gal/R23O32_Manual_0417/composite_plots_data_viz.pdf'
+        spec_range = range(27)
```

Comment:
I have spec_range defined this way to mimic the way it is defined for the other option. I had a range for the individual plots because I didn't need all of the individual spectra plotted. 
Specific paths were used because this function was written and run for a specific figure in the paper. I put the necessary paths as parameters for function now.